src/Installer.php

```diff
@@ -1,10 +1,8 @@
 <?php declare(strict_types=1);
 
-namespace Neusta\Pimcore\EditorConfigBundle;
+namespace Neusta\Pimcore\AreabrickConfigBundle\src;
 
 use Pimcore\Extension\Bundle\Installer\AbstractInstaller;
-use Pimcore\Model\Asset\Image\Thumbnail;
-use Pimcore\Model\Document\DocType;
 
 class Installer extends AbstractInstaller
```

Comment:
I don't think we need this. It does nothing.

Reply:
#metoo - removed